src/java.base/share/classes/sun/nio/ch/Interruptible.java

```diff
@@ -31,6 +31,21 @@
 
 public interface Interruptible {
 
-    public void interrupt(Thread t);
+    /**
+     * Invoked by Thread.interrupt when the given Thread is interrupted. This method
+     * is invoked while holding the Thread's interrupt lock. It will typically record
+     * that the I/O operation has been interrupted so that it can be coordinated with
+     * {@code postInterrupt} when it called after releasing the Thread's interrupt
```

Comment:
Should have been "when it is called" instead of  "when it called"

Reply:
Thanks, this is typo there.